src/NServiceBus.Persistence.CosmosDB/Transaction/Transaction.cs

```diff
@@ -2,11 +2,13 @@ namespace NServiceBus.Persistence.CosmosDB
 {
     using Features;
 
-    class Transaction : Feature
+    sealed class Transaction : Feature
     {
+        public Transaction() => Defaults(s => s.SetDefault(new TransactionInformationConfiguration()));
```

Comment:
I sort of prefer the GetOrCreate notation because this is exactly what I'm expecting it to do, unless you want to ensure you're always setting a new one

Reply:
> Just saw the discussion on the other PR :)

maybe it would be good to add it to the issue description in case someone is wondering later?

I guess this would be extremely hard to build up useful tests for to prevent accidental refactorings?

Reply:
I think it is not worth adding a test at this stage. You are welcome to do so if you have the energy... In the meantime I think we should go ahead with this.